CONTRIBUTING.rst

```diff
@@ -40,7 +40,7 @@ official dj-stripe docs, in docstrings, or even on the web in blog posts,
 articles, and such.
 
 If you are adding to dj-stripe's documentation, you can see your changes by changing
-into the ``docs`` directory, running ``make html`` (or ``make.bat html`` if you're
+into the ``docs`` directory, pip installing ``requirements.txt``, running ``make html`` (or ``make.bat html`` if you're
```

Comment:
What do you think of just adding a `tox -e docs` job?

Reply:
I like it